pkg/agent/openflow/pipeline.go

```diff
@@ -192,6 +195,30 @@ func (c *client) gatewayClassifierFlow(gatewayOFPort uint32, category cookie.Cat
 		Done()
 }
 
+// patchPortClassifierFlow generates the flow to mark IP traffic comes from the patch port and
+// are destined to local pods
+func (c *client) patchPortClassifierIPFlow(patchPort uint32, podIP net.IP) binding.Flow {
+	classifierTable := c.pipeline[classifierTable]
+	return classifierTable.BuildFlow(priorityNormal).
+		MatchInPort(patchPort).
+		MatchDstIP(podIP).
+		Action().LoadRegRange(int(marksReg), markTrafficFromPatchPort, binding.Range{0, 15}).
+		Action().ResubmitToTable(classifierTable.GetNext()).
+		Done()
+}
+
+// patchPortClassifierFlow generates the flow to mark ARP traffic comes from the patch port and
+// are destined to local pods
+func (c *client) patchPortClassifierARPFlow(patchPort uint32, podIP net.IP) binding.Flow {
```

Comment:
It looks you would like to let the peer host gateway reply ARP request in NoEncap mode, we have installed an ARP responder flow entry by default( in Encap mode), please think about if that flow is still needed.

Reply:
I was think ahead for passthrough mode ... but I can remove them...

Reply:
will keep them for subsequent changes, if u don't mind